discordjs/discordjs-commonjs/src/modules/Private/commands/Eval.js

```diff
@@ -87,12 +97,18 @@ class Eval extends Command {
         } );
     }
 
+    /**
+     * @param {String} evalString
+     */
     cleanUpToken(evalString) {
-        return evalString
-            .split(this.bot._token ? this.bot._token : this.bot.token)
-            .join('Khaaz Baguette');
+        return evalString.replace(new RegExp(this.bot.token, 'g'), 'Khaaz Baguette');
```

Comment:
will this remove all iteration of the token?

Reply:
It will indeed, it's a much cleaner way of doing it

Reply:
yeah fair enough. This was the old way of doing, it's better with replace